public/media-hunt/.gitignore

```diff
@@ -1 +1,2 @@
-.env
\ No newline at end of file
+.env
+db.json
```

Comment:
Please ensure that db.json is indeed ignored. Google around how to remove already committed file from version control and apply gitignore

Reply:
I used git rm --cached public/media-hunt/db.json to remove committed files from VC. Hopefully now it's okay.